src/bp-messages/classes/class-bp-rest-sitewide-notices-endpoint.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * BP REST: BP_REST_Sitewide_Notices_Endpoint class
+ *
+ * @package BuddyPress
+ * @since 9.0.0
+ * @deprecated 15.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+_deprecated_file(
+	basename( __FILE__ ),
+	'15.0.0',
+	'bp-members/classes/class-bp-members-notices-rest-controller.php',
+	esc_html__( 'The v1 of the BP REST API is deprecated, please use v2.', 'buddypress' )
+);
```

Comment:
I'm a but confused about having this class here. If the BP REST plugin is active, I'd expect we would get a PHP error since we would be loading the same class twice.

Reply:
We had a discussion about it, the important part is to remove the deprecated class from the class autoloader irregular map.

Reply:
I've removed the deprecated class from the the autoloader.